pkg/scheduler/util/utils.go

```diff
@@ -90,8 +93,16 @@ func MoreImportantPod(pod1, pod2 *v1.Pod) bool {
 	return GetPodStartTime(pod1).Before(GetPodStartTime(pod2))
 }
 
+const (
+	// Parameters for retrying with exponential backoff.
+	retryBackoffInitialDuration = 100 * time.Millisecond
```

Comment:
Actually, let's make this smaller. You are already sleeping the time that the server is telling you to retry.

0.1 seconds is way too much time considering that the scheduler is able to schedule ~300 pods/s if the apiserver is healthy.

maybe the factor can be 1.3

Reply:
How about updating status in goroutine instead of making backoff time smaller?
i.e. put the following line into `go func(){...}()`
https://github.com/kubernetes/kubernetes/blob/5e9a6a256e9ed7834dfcf5f4a30be015130df6e9/pkg/scheduler/schedule_one.go#L826-L833

Because if there is a connection failure, the responded error doesn't contain `RetryAfterSeconds` (since the type is not `*errors.StatusError`.) and if backoff time is small in that case, the retry may be less likely to be successful)

Reply:
> 0.1 seconds is way too much time considering that the scheduler is able to schedule ~300 pods/s if the apiserver is healthy.

Make sense. change it to `10*time.Millisecond`.

Reply:
Did you remove the comment about calling this in a routine? I think that's a good idea. But do it in the caller (the scheduler package, in this case).

Reply:
Ah, yeah, I removed my previous comments because if we do so and the parallel status update is done after starting another scheduling cycle, I guess the pod will be moved from unschedulableQ to activeQ by event handler? 
wdyt?

Reply:
Okay, It seems my concern was wrong. Any status updates are ignored and the pod won’t be moved in that case. 
https://github.com/kubernetes/kubernetes/blob/7e77252b0d5fcd3689f2f7e4a21757e53965c092/pkg/scheduler/internal/queue/scheduling_queue.go#L524

will change the scheduler pkg's implementation to update pods status in goroutine

Reply:
Should I also revert the retryBackoffInitialDuration to 0.1 seconds?

Reply:
yeah, 100ms sounds good.